### PR TITLE
store: fix timing of prewrite binlog in slow log

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What problem does this PR solve? 
The time of prewrite binlog in slow log covers the time of prewrite keys, which makes it not accurate.

### What is changed and how it works?
Fix it.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. start a cluster of master without binlog
1. start a cluster of this PR without binlog
1. compare the commit slow log with `Binlog_prewrite_time`

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch

Release note

 - Write release note for bug-fix or new feature.
Fix timing of prewrite binlog `Binlog_prewrite_time` in slow log to make it accurate.

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -1284,7 +1282,9 @@ func (c *twoPhaseCommitter) prewriteBinlog(ctx context.Context) chan *binloginfo
 		if bin.Tp == binlog.BinlogType_Prewrite {
 			bin.PrewriteKey = c.primary()
 		}
+		binlogPrewriteStart := time.Now()
 		wr := binInfo.WriteBinlog(c.store.clusterID)
+		c.getDetail().BinlogPrewriteTime = time.Since(binlogPrewriteStart)
```
Comment (lysu) [inline store/tikv/2pc.go]: maybe test case will meet race due to write/read in different goroutines

File: store/tikv/2pc.go [hunk 2]
```diff
@@ -1134,7 +1134,6 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 		}
 	}()
 
-	binlogPrewriteStart := time.Now()
 	binlogChan := c.prewriteBinlog(ctx)
```
Comment (lysu) [inline store/tikv/2pc.go]: for a slow query, maybe we are more interested in the time used to wait for this channel at line 1151~?

**Comment by shenli** [comment]
do we need to cherry-pick it to 3.1?

**Comment by jackysp** [comment]
> do we need to cherry-pick it to 3.1?

I'll check the rules with @bb7133 .

**Comment by jackysp** [comment]
PTAL @lysu

**Pull request merged**